.travis.yml

```diff
@@ -1,55 +1,62 @@
+sudo: required
+
 language: python
 
+dist: trusty
+
+notifications:
+    email: false
```

Comment:
I think we do want notifications. Any specific reason you think this is not a good idea?